jade/distribution.yaml

```diff
@@ -3650,6 +3650,51 @@ repositories:
       url: https://github.com/ros-visualization/rqt_robot_plugins.git
       version: master
     status: developed
+  rsv_balance:
+    doc:
+      type: git
+      url: https://github.com/robosavvy/rsv_balance.git
+      version: master
+    release:
+      packages:
+      - rsv_balance_description
+      - rsv_balance_msgs
+      url: https://github.com/robosavvy/rsv_balance.git
```

Comment:
Thank you,
It was unclear from the tutorial how could I index my packages for wiki only, without binary release.

As far as I can deduce, should I delete the lines pertaining "release:" from my entries?
Such like: cd9f2b68341b6c565cb96976f75f9fd605b2533d